cinema/models.py

```diff
@@ -50,9 +50,11 @@ class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
-    genres = models.ManyToManyField(Genre)
-    actors = models.ManyToManyField(Actor)
-    image = models.ImageField(null=True, upload_to=movie_image_file_path)
+    genres = models.ManyToManyField(Genre, blank=True)
+    actors = models.ManyToManyField(Actor, blank=True)
```

Comment:
The `genres` and `actors` fields in the `Movie` model are correctly set as many-to-many relationships with `Genre` and `Actor` models, respectively. The `blank=True` option allows these fields to be optional, which is appropriate for this context.